helper/kvdb/kvdb.go

```diff
@@ -0,0 +1,20 @@
+package kvdb
+
+type KVBatch interface {
+	Set(k, v []byte)
+	Write()
```

Comment:
Must returns error when failed. We should not "swallow" any failure.

Reply:
@DarianShawn 9d3272f

Reply:
Got it. 🙌